package/kernel/linux/modules/tpm.mk

```diff
@@ -0,0 +1,36 @@
+TPM_MENU:=TPM
+
+define KernelPackage/tpm
```

Comment:
kernel: add tpm support -> 
It would be good to mention addition of "kmod-tpm support" in the commit message.